crates/common_enums/src/enums.rs

```diff
@@ -2210,6 +2210,35 @@ pub enum ScaExemptionType {
     TransactionRiskAnalysis,
 }
 
+#[derive(
+    Clone,
+    Copy,
+    Debug,
+    Default,
+    Eq,
+    PartialOrd,
+    Ord,
+    Hash,
+    PartialEq,
+    serde::Deserialize,
+    serde::Serialize,
+    strum::Display,
+    strum::VariantNames,
+    strum::EnumIter,
+    strum::EnumString,
+    ToSchema,
+)]
+#[router_derive::diesel_enum(storage_type = "text")]
+#[serde(rename_all = "snake_case")]
+#[strum(serialize_all = "snake_case")]
+/// Describes the channel through which the payment was initiated.
+pub enum PaymentChannel {
+    #[default]
+    Ecommerce,
+    MailOrder,
+    TelephoneOrder,
```

Comment:
```suggestion
    TelephoneOrder,
    #[serde(untagged)]
    #[strum(default)]
    Other(String),
```

This will avoid failure of parsing the new enum variant during the staggered release